mailbox/event/json/src/main/scala/org/apache/james/event/json/EventSerializer.scala

```diff
@@ -115,16 +139,31 @@ private object ScalaConverter {
     totalDeletedSize = event.getTotalDeletedSize,
     mailboxId = event.getMailboxId)
 
+  private def toScala(event: JavaAdded): DTO.Added = DTO.Added(
+    sessionId = event.getSessionId,
+    user = event.getUser,
+    path = DTO.MailboxPath.fromJava(event.getMailboxPath),
+    mailboxId = event.getMailboxId,
+    added = event.getAdded.asScala.map(entry => entry._1 -> MetaDataDTO.MessageMetaData.fromJava(entry._2)).toMap)
+
+  private def toScala(event: JavaMessageMoveEvent): DTO.MessageMoveEvent = DTO.MessageMoveEvent(
+    user = event.getUser,
+    previousMailboxIds = event.getMessageMoves.getPreviousMailboxIds.asScala.toList,
```

Comment:
you probably don't need toList anymore, do you ?

Reply:
A Set is not a Seq so no, this is needed

Reply:
I ended up choosing Iterable as JavaSet.asScala returns an Iterable. The rest of the wiring is easy...